modules/ROOT/pages/policies-policy-overview.adoc

```diff
@@ -0,0 +1,87 @@
+
+Some policies, such as the token enforcement policy, allow propagation of headers, including the information related to the token. This token is used by the implementation service or by the API itself.
+
+//Eva--add info about the generic policy scope Add an example or link here..Fede Balbi?
+
+
+== Policy Size
+
+The size of a policy varies based on its dependencies. The first time Mule Runtime polls for policies it will take longer to fetch them. However, after policies are retrieved, they are cached on the file system. This ensures offline availability and also reduces latency to fetch policies.
+//Add more information about the policy size. we have two options here, either hide the policy size, or clarify that this doesn't happen for cloudhub or runtime fabric--chirag
+
+
+
+== Policy Level
+
+Policies implement an additional level of granularity, where access is controlled based on a criteria. In this level of granularity, policies are applied to only those request that match the criteria.  These polices are known as resource-level policies. All policies, except the Cross-Origin Resource Sharing (CORS) policy, are resource-level.
+
+Resource level policy support includes RAML and HTTP-based APIs (since Mule 4). A policy applied at a resource level affects all HTTP methods, for example, PUT, GET, POST, or selected HTTP methods, within the resource.
+
+image::methods-resources.png[]
+
+You can apply multiple conditions to filter your resources and HTTP methods. The URI template regex can be applied to one, many, or all the methods that your API has. The uses for resource level policies are limitless. Listed below are a few examples of how you might implement the resource-level policies:
+
+* Applying policies to specific resources
+* Securing a subset of an API
+* Setting different limits on resources
+
+If you have a RAML/OAS spec attached to your API, click Preview Resource Matching to check which resources your filters affect.
+
+== Policies and SLAs
+
+You can configure specific policies to adhere to predefined SLAs. You create tiers of access depending on one of the following SLA tiers applied to the application:
+
+* A tier that limits requests to three per minute. No approval required.
+* A tier that limits requests to five per minute and requires API Versions Owner approval of the application that wants to access the API.
+
+With SLAs, you limit access to one API resource. Access to other resources are unlimited. When an app attempts to consume the protected resource, the policy is enforced. The request must include the expected user name and password. Repeated calls within SLA limits from the app to the API succeed; others fail.
+
+//SLA tiers should link to all the documentation on contracts, request api access and those flows.
+
+//== Policy Lifecycle
```

Comment:
There is one section that I think we are missing, we can add it later on, but to keep track of it:

- Policies and contracts: this is because some policies require a valid contract between the API and the application in order to grant access to the API or API resource. For instance: client id enforcement, Token Enforcement Policies. 

@priyaayyar could you maybe add a Jira and I can help you providing the content?

Reply:
Done! Thanks!